content/docs/engine-dev/_index.md

```diff
@@ -61,6 +61,15 @@ O3DE asset system is a key part of managing and loading assets in a project. Thi
 |-|-|
 | [Metadata Relocation](./assets/metadata) | Learn about how the Metadata Relocation system functions, where the code is, and which file types may or may not be supported. |
 
+## Scene Pipeline
+
+Content in scene files, like `fbx` format files, are core to most O3DE projects. This section of the Engine Developer Guide provides details on the scene pipeline to help developers extend or modify the system for their needs.
```

Comment:
nit: Add line to cover what AssImp is ie `O3DE uses the Open Asset Import Library as the foundation of content importing.